### PR TITLE
Updating tests to remain the same with NJOY21.

### PR DESCRIPTION
**Pull request by jlconlin**
In an effort to keep the tests between NJOY2016 and NJOY21 the same, I've updated a few of the input decks here in NJOY2016.

### CODE REVIEW

File: tests/61/input [hunk 1]
```diff
@@ -7,7 +7,7 @@
  acer
   0 61  0 71 72/             card1
   7 1 1 0.90/                card2
-  /                          card3
+  'e80 h-h2o' /              card3
```
Comment (whaeck) [inline tests/61/input:10]: iopt=7 allows you to leave card 3 empty. That's legal. Does lipservice complain about this?

File: tests/CMakeLists.txt [hunk 2]
```diff
@@ -1,5 +1,7 @@
 set( RESOURCES "${CMAKE_CURRENT_SOURCE_DIR}/resources" )
 
+find_package( Python3 3.5 REQUIRED )
```
Comment (whaeck) [inline tests/CMakeLists.txt:3]: Did you remove this from the main CMakeLists.txt?

Develop should be updated to the latest master too. I don't think this is done yet.

**Review by whaeck — CHANGES_REQUESTED** [review]

File: tests/CMakeLists.txt (hunk 2 above)
Comment (jlconlin) [reply tests/CMakeLists.txt:3]: I did not remove this from the top level. I meant to, but forgot.

File: tests/61/input (hunk 1 above)
Comment (jlconlin) [reply tests/61/input:10]: lipservice complains if there is nothing there. We could update lipservice to give a default value. That would be more consistent with the other ways we've been doing things.

Comment (jlconlin) [reply tests/61/input:10]: An empty string works with lipservice, but no string does not.

**Pull request closed without merging**